server/database/models/rating.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
+module.exports = (sequelize, DataTypes) => {
+  const Rating = sequelize.define('Rating', {
+    rating: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
```

Comment:
Is `userId` not suppose to reference `Users` table here or is there a reason why you decided to do this

Reply:
There should be an association between the user and the ratings table